packages/player/src/internal/handlers/set-event-sender.ts

```diff
@@ -0,0 +1,13 @@
+import type { EventSender } from '@tidal-music/common';
+
+import { eventSenderStore } from '../index';
+
+/**
+ * Set the credentials provider TIDAL Player SDK should use for getting
+ * session information
+ *
+ * @param {CredentialsProvider} newCredentialsProvider
```

Comment:
is `EventSender` a `CredentialsProvider`?

Reply:
That's fixed in the PR pointing to this one too: https://github.com/tidal-music/tidal-sdk-web/pull/92/files#diff-82bfd180438fbde440d4fcada5f089f39b883341b11294ca7b1d0db2b8946d55

Got a bit dissorted since I reverted the type setting in common. Sorry for the confusion.